frontend/src/app/modules/auth/sign-in/sign-in.component.ts

```diff
@@ -63,4 +74,24 @@ export class SignInComponent {
     get password(): FormControl {
         return this.form.get('password') as FormControl;
     }
+
+    ClearErrors() {
+        this.isChanged = false;
+    }
+
+    GetErrorMessageByKey(id: string) {
+        return Object.entries(validationErrorMessage).find(([t]) => t === id)?.[1];
+    }
+
+    CreateCondition(contr: FormControl, cond: string) {
```

Comment:
Create -> Check or something that way

Reply:
And name vars in full name, control and condition

Reply:
fixed